Recruit-Training/Advanced-Recruit-Training/README.adoc

```diff
@@ -26,22 +26,36 @@ You'll have to set the appropriate command line arguments in `Projects > Build &
 
 == Objective
 
-The goal of this project is to give new recuits a good starting point to learn the required skills to work on the other projects. 
-If you ever need any help with how to use Qt's functions, a good start would be to look at http://doc.qt.io/qt-5.6/reference-overview.html[Qt's reference manual]
+The goal of this project is to give new recruits a good starting point to learn the required skills to work on the other projects. 
+If you ever need any help with how to use Qt's functions, a good start would be to look at http://doc.qt.io/qt-5.6/reference-overview.html[Qt's reference manual]. 
+Another useful resource can be found at https://doc.qt.io/qt-5/qtcore-module.html#namespaces. 
+This lists and explains how to implement Qt core C++ classes, some of which (i.e. QTime) will need to be used to complete the training tasks. 
+Refer to the development resources folder for the style guide. 
+In this document you can find the stylistic conventions that are to be used throughout the program.
 
 === Tasks to Complete
 
- *  Complete the error checking in `LogFileReader::parseLine`. 
- The proper format of each line in batData.csv should be hh:mm:ss:zzz, so any other differences should be caught. 
- It should be able to catch all the errors in batDataFailure.csv.
+ *  Complete the error checking in `LogFileReader::parseLine`. The proper format of each line in batData.csv should be hh:mm:ss:zzz, so any other differences should be caught. 
+It should be able to catch all the errors in batDataFailure.csv. 
  *  Implement the functions of BatteryStateOfChargeService. 
- See the notes below on how to calculate the values.
- *  Print out the time until depleted or charged (found in BatteryStateOfChargeService) in `BatteryStateDisplayService::handleBatteryDataReceived`.
-
-=== Values to Note for BatteryStateOfChargeService
-
- *  The battery is charging when the battery current is negative, depleting when it is positive.
- *  The initial amount of Amphours used would be the Battery's Amphour capacity(Ah) multiplied by it's state of charge(%).
- *  The change in amphours used between two data points would be the average current(A) multiplied by the difference in time(h)
- *  The time until charge would be the Amphours used(Ah) divided by the current (A). 
- Similarily, the time until depletion would be the remaining amphours divided by the current.    
+The instructions for how to calculate the required values can be found in the values and calculations section below.
+ *  `BatteryStateDisplayService::handleBatteryDataReceived` can be found in BatteryStateOfChargeService. 
+In this function, the time until the battery is charged or depleted needs to be printed.
+ *  In order to print newly calculated values, the variables must be updated. 
+There are many different ways to do this. 
+This should be implemented within the addData function which can be found in BatteryStateOfChargeService.
```

Comment:
I think this is the one that had a bug, can see a fix for this in two ways:
1) Find the faulty data and change the values
2) It's a feature not a bug! Mention that there's a problem with the values, and tell them to try to find what the problem is